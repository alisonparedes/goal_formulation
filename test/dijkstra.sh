# distance to base
python ../src/dijkstra.py ../test/initial_state_01.txt B
python ../src/dijkstra.py ../test/initial_state_02.txt B
python ../src/dijkstra.py ../test/initial_state_03.txt B
python ../src/dijkstra.py ../test/initial_state_04.txt B
python ../src/dijkstra.py ../test/initial_state_05.txt B
python ../src/dijkstra.py ../test/initial_state_06.txt B
python ../src/dijkstra.py ../test/initial_state_07.txt B
python ../src/dijkstra.py ../test/initial_state_08.txt B

# distance to food
python ../src/dijkstra.py ../test/initial_state_08.txt F_0_0
python ../src/dijkstra.py ../test/initial_state_09.txt F_0_0
python ../src/dijkstra.py ../test/initial_state_10.txt F_0_0
python ../src/dijkstra.py ../test/initial_state_11.txt F_0_0
# 12 doesn't contain food
python ../src/dijkstra.py ../test/initial_state_13.txt F_0_0
python ../src/dijkstra.py ../test/initial_state_14.txt F_0_0
# 15 doesn't contain food
python ../src/dijkstra.py ../test/initial_state_16.txt F_0_0
# more food
python ../src/dijkstra.py ../test/initial_state_17.txt F_0_0
python ../src/dijkstra.py ../test/initial_state_18.txt F_0_0
python ../src/dijkstra.py ../test/initial_state_19.txt F_0_0
python ../src/dijkstra.py ../test/initial_state_20.txt F_0_0
python ../src/dijkstra.py ../test/initial_state_21.txt F_0_0
python ../src/dijkstra.py ../test/initial_state_22.txt F_0_0
python ../src/dijkstra.py ../test/initial_state_23.txt F_0_0
python ../src/dijkstra.py ../test/initial_state_24.txt F_0_0
# base again
python ../src/dijkstra.py ../test/initial_state_17.txt B
python ../src/dijkstra.py ../test/initial_state_18.txt B
python ../src/dijkstra.py ../test/initial_state_19.txt B
python ../src/dijkstra.py ../test/initial_state_20.txt B
python ../src/dijkstra.py ../test/initial_state_21.txt B
python ../src/dijkstra.py ../test/initial_state_22.txt B
python ../src/dijkstra.py ../test/initial_state_23.txt B
python ../src/dijkstra.py ../test/initial_state_24.txt B
