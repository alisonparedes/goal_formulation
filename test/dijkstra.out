initial_state: ../test/initial_state_01.txt
goal: B
21 F?
10 ?b

initial_state: ../test/initial_state_02.txt
goal: B
21 F?
10 HB

initial_state: ../test/initial_state_03.txt
goal: B
21 H?
10 ?B

initial_state: ../test/initial_state_04.txt
goal: B
21 ??
10 HB

initial_state: ../test/initial_state_05.txt
goal: B
21 ?$
10 ?B

initial_state: ../test/initial_state_06.txt
goal: B
21 $?
10 ?B

initial_state: ../test/initial_state_07.txt
goal: B
21 F?
10 HB

initial_state: ../test/initial_state_08.txt
goal: B
21 ??
10 ?*

initial_state: ../test/initial_state_08.txt
goal: F_0_0
01 ??
12 ?*

initial_state: ../test/initial_state_09.txt
goal: F_0_0
01 F?
12 ?b

initial_state: ../test/initial_state_10.txt
goal: F_0_0
01 FH
12 ?B

initial_state: ../test/initial_state_11.txt
goal: F_0_0
01 F?
12 HB

initial_state: ../test/initial_state_13.txt
goal: F_0_0
0X F#
12 ?b

initial_state: ../test/initial_state_14.txt
goal: F_0_0
0X F#
12 HB

initial_state: ../test/initial_state_16.txt
goal: F_0_0
0X F#
12 ?*

initial_state: ../test/initial_state_17.txt
goal: F_0_0
0X67 F#??
1X56 ?#??
2345 ????
3X56 ?#?b

initial_state: ../test/initial_state_18.txt
goal: F_0_0
0X67 F#H?
1X56 ?#??
2345 ????
3X56 ?#?B

initial_state: ../test/initial_state_19.txt
goal: F_0_0
0X67 F#??
1X56 ?#??
2345 ????
3X56 ?#?*

initial_state: ../test/initial_state_20.txt
goal: F_0_0
0X67 F#??
1X56 ?#$?
2345 ????
3X56 ?#?B

initial_state: ../test/initial_state_21.txt
goal: F_0_0
0X67 F#??
1X56 ?#$?
2345 ????
3X56 ?#?B

initial_state: ../test/initial_state_22.txt
goal: F_0_0
0X67 F#??
1X56 ?#??
2345 ????
3X56 H#?B

initial_state: ../test/initial_state_23.txt
goal: F_0_0
0X67 F#??
1X56 ?#??
2345 ??$?
3X56 ?#?B

initial_state: ../test/initial_state_24.txt
goal: F_0_0
0X67 F#??
1X56 ?#?H
2345 ????
3X56 ?#?B

initial_state: ../test/initial_state_17.txt
goal: B
6X43 F#??
5X32 ?#??
4321 ????
5X10 ?#?b

initial_state: ../test/initial_state_18.txt
goal: B
6X43 F#H?
5X32 ?#??
4321 ????
5X10 ?#?B

initial_state: ../test/initial_state_19.txt
goal: B
6X43 F#??
5X32 ?#??
4321 ????
5X10 ?#?*

initial_state: ../test/initial_state_20.txt
goal: B
6X43 F#??
5X32 ?#$?
4321 ????
5X10 ?#?B

initial_state: ../test/initial_state_21.txt
goal: B
6X43 F#??
5X32 ?#$?
4321 ????
5X10 ?#?B

initial_state: ../test/initial_state_22.txt
goal: B
6X43 F#??
5X32 ?#??
4321 ????
5X10 H#?B

initial_state: ../test/initial_state_23.txt
goal: B
6X43 F#??
5X32 ?#??
4321 ??$?
5X10 ?#?B

initial_state: ../test/initial_state_24.txt
goal: B
6X43 F#??
5X32 ?#?H
4321 ????
5X10 ?#?B

